react-app/src/components/SplashPage.js

```diff
@@ -1,25 +1,59 @@
 import React, { Component } from "react";
-
-import { Sprite } from "react-pixi-fiber";
+import { Sprite, Container, Text } from "react-pixi-fiber";
 import * as PIXI from "pixi.js";
 import splashPage from "./images/monster.png";
+import StartButton from "./StartButton";
 
 const centerAnchor = new PIXI.Point(0, 0);
 
 class SplashPage extends Component {
+  constructor(props){
+    super(props);
+    this.hoverOver = this.hoverOver.bind(this);
+    this.hoverOff = this.hoverOff.bind(this);
+    this.state = {hoverOverStyle : this.props.style}
```

Comment:
Setting state from props is usually a sign something is amiss. See if you and Quincey can come up with a simpler way to handle the style. Happy to help provide more info if you need!